components/tools/OmeroWeb/omeroweb/webclient/tree.py

```diff
@@ -59,7 +59,8 @@ def parse_permissions_css(permissions, ownerid, conn):
     permissionsCss = [r for r in restrictions if permissions.get(r)]
     if ownerid == conn.getUserId():
         permissionsCss.append("isOwned")
-    if ownerid == conn.getUserId() or conn.isAdmin():
+    if (ownerid == conn.getUserId() or
+            "Chgrp" in conn.getEventContext().adminPrivileges):
```

Comment:
Looks good. For unification of our codebase, can this be adjusted to use the enums as in https://github.com/openmicroscopy/openmicroscopy/pull/5376/files#diff-4c592a4ec58d6f4830c88b81f8aa77a3R66